### DISCUSSION TITLE
Loading plots takes long time

### DISCUSSION THREAD

**Discussion by cccat6** [Chia-Network/chia-blockchain#6380] [Support]
When I getting more and more plots and mount them on a single machine. It will take a long long time to load all these plots. (Searching directories)
When I get few TiBs, it may take few seconds. But now I got 1.5PiB and it takes tens of minutes to load.
I monitor the read file requests from Chia processes. It will only read one file at a time. It should be able to load all files into the cache parallel.
Will it be fixed? very appreciate!

**Comment by joelrb** [comment]
> When I getting more and more plots and mount them on a single machine. It will take a long long time to load all these plots. (Searching directories)
> When I get few TiBs, it may take few seconds. But now I got 1.5PiB and it takes tens of minutes to load.
> I monitor the read file requests from Chia processes. It will only read one file at a time. It should be able to load all files into the cache parallel.
> Will it be fixed? very appreciate!

Maybe it's time for you to to Farm on many machines? 1.5PiB I think is too much for one machine.

https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

Use the same keys you have in your main machine and your plots will be recognized in another machine/s. Just distribute your plots to maybe 2-3 machines.

**Reply by cccat6** [reply]
Actually, when I am farming, CPU utilization is always below 10%. It is a pretty light load. After one hour of waiting for Searching directories, it back to work.

**Reply by chiavailable** [reply]
It looks like it will be a bad connection. Damage cable, bad adapter, or your system using 2.0 usb or older?

**Reply by cccat6** [reply]
No, but it is NFS. However, I cached everything like the head and tail and metadata locally in memory. It is not likely bottlenecked on the connection speed.
I got more than 15,000 plots and I think that takes time to load in a single thread.

**Comment by BrandtH22** [comment]
@cccat6 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
